packages/web/components/transactions/transaction-row.tsx

```diff
@@ -121,22 +121,22 @@ const TokenConversion: FunctionComponent<
     Transaction["tokenConversion"]
   >
 > = ({ status, tokenIn, tokenOut, effect }) => (
-  <>
-    <div className="flex w-1/3 items-center justify-end gap-4 md:hidden md:w-auto">
+  <div className="flex w-2/3 items-center justify-end gap-4 md:w-1/2">
+    <div className="flex w-48 items-center justify-end gap-4 md:hidden">
       <div className="flex flex-col text-right md:hidden">
         {tokenIn.value && (
           <div
-            className={classNames("text-subtitle1", {
-              "text-osmoverse-400": status === "pending",
+            className={classNames("subtitle1", {
+              "text-osmoverse-480": status === "pending",
               "text-osmoverse-100": status === "success",
               "text-rust-400": status === "failed",
             })}
           >
-            - ${Number(tokenIn.value.toDec().toString()).toFixed(2)}
+            {formatPretty(tokenIn.amount, { maxDecimals: 6 })}
           </div>
         )}
-        <div className="text-body2 text-osmoverse-400">
-          {formatPretty(tokenIn.amount, { maxDecimals: 6 })}
+        <div className="body2 text-osmoverse-400">
+          - ${Number(tokenIn?.value?.toDec().toString()).toFixed(2) || 0}
```

Comment:
Perhaps use `tokenIn?.value?.symbol` instead of hardcoding USD, will help us if one day we can support other fiat currencies

Please make sure elsewhere we are not hardcoding the symbol.

Also separate- an issue I've noticed with this approach is it removes commas from 1k+ values (which nass has opposed)

Reply:
Perhaps we need to look into using our `formatPretty`, or fixing it to trail with two 00s always with prices (or however many 0s the price locale uses, USD is 2 for example).